test/ontarget/S32K146EVB/include/system_S32K146.h

```diff
@@ -39,12 +38,13 @@
  */
 
 #ifndef SYSTEM_S32K146_H_
-#define SYSTEM_S32K146_H_                        /**< Symbol preventing repeated inclusion */
+#define SYSTEM_S32K146_H_ /**< Symbol preventing repeated inclusion */
```

Comment:
There should be two spaces before the comment. Perhaps clang-format is misconfigured?

Reply:
Hmm. Let me see if this is something clang-format can do...

Reply:
It definitely can (at least v8.0 can).

Reply:
```
SpacesBeforeTrailingComments: 2
```
Is already in the config. I'm not sure why it didn't honor it for this line?

Reply:
Ah, I see it now. This only works for // comments.

Reply:
Sad.

Reply:
I tried to manually clean this up a bit. It is just test code. I'll be more fastidious in the libuavcan headers.

Reply:
Okay